game-core/src/main/java/games/strategy/engine/lobby/server/userDB/DBUser.java

```diff
@@ -1,140 +0,0 @@
-// CHECKSTYLE-OFF: PackageName
-// move this class to lobby.common upon next lobby-incompatible release; it is shared between client
```

Comment:
We should definitely search for "incompatible" before pushing a new release

Reply:
Yeah! My 2 cents, I think we probably should work on an incompatible very quickly after this upcoming release. The efficiency improvement and liberty to make change is very significant. @prastle points out that for players the incompatible update is not quite as impactful as one would think (it still hurts, but the people claiming to abandon TripleA after incompatible are few and mostly being loud; yet with that said we should try to make upgrades as painless as we can).